analyses/summary/helpers.go

```diff
@@ -45,16 +45,22 @@ func getResultSummary(analysedResult driver.GetResponseType) *StackSummary {
 func processVulnerabilities(analysedDeps []driver.AnalysedDepsType) ProcessVulnerabilities {
 	processedData := &ProcessVulnerabilities{}
 	for _, dep := range analysedDeps {
-		public := len(dep.PublicVulnerabilities)
-		private := len(dep.PrivateVulnerabilities)
-		if public+private > 0 {
+		publicVul := len(dep.PublicVulnerabilities)
+		privateVul := len(dep.PrivateVulnerabilities)
+		if publicVul+privateVul > 0 {
 			processedData.DirectVulnerableDependencies++
 		}
 		processedData.TotalTransitives += len(dep.Transitives)
 		processedData.Severities = getSeverity(dep.PublicVulnerabilities, processedData.Severities)
 		processedData.Severities = getSeverity(dep.PrivateVulnerabilities, processedData.Severities)
-		processedData.PublicVul += public
-		processedData.PrivateVul += private
+		for _, transVul:= range dep.VulnerableDependencies{
```

Comment:
Your pr title must convey this subtle functionality change. Now you are including transitive vulnerabilities right?

Reply:
Please make changes in title whatever you feel appropriate